README.md

```
@@ -774,8 +774,24 @@ To regenerate import URL if needed:
  global.cattle.clusterId: "c-m-pbrcfglw" # ← REPLACE THIS
 ```
 
+- **Alerting Configuration:**
+  Alerting is part of cluster monitoring, where alert notifications are sent to the configured email or slack channel.
+   * `<slack-channel-name>` → slack channel name configured for alert notifications.
+   * `<slack-api-url>` → slack api url configured for alert notifications.
+   * `<env-name>` → provide the cluster name.
+
+> **Note:** 
+> * Creating slack incoming webhook [Link](https://docs.slack.dev/messaging/sending-messages-using-incoming-webhooks/),
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)